app/database_etl/etl_main.py

```diff
@@ -203,22 +220,32 @@ def create_bridge_tables(original_data, multi_select_tables):
 def load_postgres_tables(tables_dict, engine):
     # Load dataframes into postgres tables
     for table_name, table_value in tables_dict.items():
-        table_value.to_sql(table_name,
-                           schema='public',
-                           con=engine,
-                           if_exists='append',
-                           index=False)
-    return
-
-
-def drop_old_entries():
-    all_tables = [DashboardSource, ResearchSource, City, State, TestManufacturer, AntibodyTarget, Country,
-                  CityBridge, StateBridge, TestManufacturerBridge, AntibodyTargetBridge]
-    with db_session() as session:
-        for table in all_tables:
-            # Drop record if it was not added during the current run
-            session.query(table).filter(table.created_at != CURR_TIME).delete()
-        session.commit()
+        try:
+            table_value.to_sql(table_name,
+                               schema='public',
+                               con=engine,
+                               if_exists='append',
+                               index=False)
+        except (SQLAlchemyError, ValueError) as e:
+            # Send error email
+            logging.error(e)
+            # TODO: send slack message with error (include the table name that failed)
+            # Break out of function and return 'failure'
+            return 'failure'
```

Comment:
nit: True/False vs 'success' and 'failure'?

Reply:
sure